ui/packages/ace/lib/start.js

```diff
@@ -1,14 +1,19 @@
 const open = require('opn')
 const webpack = require('webpack')
 const WebpackDevServer = require('webpack-dev-server')
+const clc = require('cli-color')
 
 const webpackConfig = require('./webpack.config')
+const warn = clc.yellowBright.blink
 
 module.exports = ({ args, pkg }) => {
   const config = webpackConfig({
     env: process.env.NODE_ENV || args.env || 'development',
     main: pkg.main,
-    alias: pkg.alias
+    alias: pkg.alias,
+    auth: args.auth ||
+      console.log(warn('WARNING: using default basic auth (admin:admin)! See options for how to override this.')) ||
```

Comment:
❓ This syntax is new to me. I am guessing `console.log` or `warn` evaluates/returns a falsey value, so it falls through to the third case?

Reply:
I don't know if it has a special name, but I've had success searching for `Javascript double pipe` and getting [results](https://stackoverflow.com/questions/2802055/what-does-the-construct-x-x-y-mean/2802064#2802064) about it in the past

Reply:
I think it evaluates to `undefined`

Reply:
![image](https://user-images.githubusercontent.com/8922497/38629111-fa47d40a-3d67-11e8-870d-64a9f6a65adf.png)